NebulaNetwork/PlayerManager.cs

```diff
@@ -214,24 +213,15 @@ public void SendPacketToOtherPlayers<T>(T packet, Player sender) where T : class
             }
         }
 
-        public Player PlayerConnected(NebulaConnection conn)
+        public NebulaPlayer PlayerConnected(NebulaConnection conn)
         {
             //Generate new data for the player
             ushort playerId = GetNextAvailablePlayerId();
 
             Float3 PlayerColor = new Float3(Config.Options.MechaColorR / 255, Config.Options.MechaColorG / 255, Config.Options.MechaColorB / 255);
-            PlanetData birthPlanet = GameMain.galaxy.PlanetById(GameMain.galaxy.birthPlanetId);
-            PlayerData playerData;
-            if (LocalPlayer.GS2_GSSettings != null)
-            {
-                playerData = new PlayerData(playerId, -1, PlayerColor, position: new Double3(birthPlanet.uPosition.x, birthPlanet.uPosition.y, birthPlanet.uPosition.z));
-            }
-            else
-            {
-                playerData = new PlayerData(playerId, -1, PlayerColor);
-            }
+            PlayerData playerData = new PlayerData(playerId, -1, PlayerColor);
```

Comment:
This will break compatibility with Galactic Scale 2. In GS2 birth system can be not at zero. Because of that you need to set position. Without GS2 it should default to (0,0,0)
```cs
Float3 PlayerColor = new Float3(Config.Options.MechaColorR / 255, Config.Options.MechaColorG / 255, Config.Options.MechaColorB / 255);
PlanetData birthPlanet = GameMain.galaxy.PlanetById(GameMain.galaxy.birthPlanetId);
PlayerData playerData = new PlayerData(playerId, -1, PlayerColor, position: new Double3(birthPlanet.uPosition.x, birthPlanet.uPosition.y, birthPlanet.uPosition.z));
```

Reply:
Done in cc2ff2f